airflow/contrib/kubernetes/kube_client.py

```diff
@@ -48,4 +48,10 @@ def _load_kube_config(in_cluster, cluster_context, config_file):
 def get_kube_client(in_cluster=conf.getboolean('kubernetes', 'in_cluster'),
                     cluster_context=None,
                     config_file=None):
+    if cluster_context is None \
+       and conf.get('kubernetes', 'cluster_context'):
+        cluster_context = conf.get('kubernetes', 'cluster_context')
```

Comment:
```diff
+    if not in_cluster:
+        if cluster_context is None
+            cluster_context = conf.get('kubernetes', 'cluster_context', fallback=None)
+        if config_file is None:
+            config_file = conf.get('kubernetes', 'config_file', fallback=None)
     return _load_kube_config(in_cluster, cluster_context, config_file)
```

Reply:
Thanks @ashb. Its my Bad, I misinterpreted your comments. Thanks again for reviewing this.
Incorporated your comments